### PR TITLE
Switch status detach feature

### PR DESCRIPTION
**Pull request by cinhcet**

### CODE REVIEW

File: nodes/ui_switch.html [hunk 1]
```diff
@@ -117,6 +126,10 @@
         <label style="width:auto" for="node-input-passthru"><i class="fa fa-arrow-right"></i> If <code>msg</code> arrives on input, pass through to output: </label>
         <input type="checkbox" checked id="node-input-passthru" style="display:inline-block; width:auto; vertical-align:top;">
     </div>
+    <div class="form-row form-row-decouple">
+        <label style="width:auto" for="node-input-decouple"><i class="fa fa-arrow-right"></i> If enabled, the switch shows the state of the input, not the output: </label>
+        <input type="checkbox" id="node-input-decouple" style="display:inline-block; width:auto; vertical-align:top;">
```
Comment (dceejay) [inline nodes/ui_switch.html]: Would really prefer this to be a select if possible

**Comment by dceejay** [comment]
Looks good. though must admit it does look and feel odd pressing the switch and nothing happening... until the feedback comes round... which as others have noted can be some time on a slow network - but hey... that's why we make it optional :-)

**Comment by cinhcet** [comment]
its now a select (I had to change the value to a string, is there a better way?!?)

Do you think of a better solution like discussed in https://groups.google.com/forum/#!topic/node-red/f3aVMDkIGiQ 
The problem I see is how to detect the case that the feedback has not come properly

**Comment by dceejay** [comment]
Well it does "seem" to do the right thing... if you press it and it doesn't get feedback - and press it again - it resends the value you expect.... ie it doesn't toggle back. So I think it's OK... It's up to the app developer to make sure there is feedback if using this mode of operation.

Happy to accept it as-is if you are happy.

**Comment by cinhcet** [comment]
I would be happy, yes :-)

Unfortunately, I found a little problem on mobile: When swiping over the switch, I can move it without changing its value. But as far as I can see, this was also the case before?

**Comment by dceejay** [comment]
I think a bit of
    ng-swipe-right="$event.stopPropagation();"
    ng-swipe-left="$event.stopPropagation();"
in switch.html may fix it (similar to slider.html)

**Comment by cinhcet** [comment]
this code is already there. I just updated the browser on that device and now the bug has gone... sorry for the confusion :-P

**Comment by dceejay** [comment]
@cinhcet  Thanks for this...

**Pull request merged**